index.html

```diff
@@ -1,4254 +1,68 @@
-<li><a href="https://youtu.be/XNXJtxkUKeY">Automation of Everything - How To Combine Argo Events, Workflows &amp; Pipelines, CD, and Rollouts</a></li>
-<li><a href="https://youtu.be/UMaivwrAyTA">Argo Workflows and Pipelines - CI/CD, Machine Learning, and Other Kubernetes Workflows</a></li>
-<li><a href="https://medium.com/@marekermk/provisioning-argo-on-openshift-with-ansible-and-kustomize-340a1fda8b50">Argo Ansible role: Provisioning Argo Workflows on OpenShift</a></li>
-<li><a href="http://bit.ly/30YNIvT">Argo Workflows vs Apache Airflow</a></li>
-<li><a href="https://medium.com/@bouwe.ceunen/ci-cd-with-argo-on-kubernetes-28c1a99616a9">CI/CD with Argo on Kubernetes</a></li>
-<li><a href="https://github.com/terrytangyuan/distributed-ml-patterns">Distributed Machine Learning Patterns from Manning Publication</a></li>
-<li><a href="https://admiralty.io/blog/running-argo-workflows-across-multiple-kubernetes-clusters/">Running Argo Workflows Across Multiple Kubernetes Clusters</a></li>
-<li><a href="https://www.anaconda.com/blog/developer-blog/open-source-model-management-roundup-polyaxon-argo-and-seldon/">Open Source Model Management Roundup: Polyaxon, Argo, and Seldon</a></li>
-<li><a href="https://www.interline.io/blog/scaling-openstreetmap-data-workflows/">Producing 200 OpenStreetMap extracts in 35 minutes using a scalable data workflow</a></li>
-<li><a href="http://dev.matt.hillsdon.net/2018/03/24/argo-integration-review.html">Argo integration review</a></li>
-<li>TGI Kubernetes with Joe Beda: <a href="https://www.youtube.com/watch?v=M_rxPPLG8pU&amp;start=859">Argo workflow system</a></li>
-</ul>
-<h2 id="project-resources">Project Resources<a class="headerlink" href="#project-resources" title="Permanent link">&para;</a></h2>
-<ul>
-<li><a href="https://github.com/argoproj">Argo Project GitHub organization</a></li>
-<li><a href="https://argoproj.github.io/">Argo Website</a></li>
-<li><a href="https://argoproj.github.io/community/join-slack">Argo Slack</a></li>
-</ul>
-<h2 id="security">Security<a class="headerlink" href="#security" title="Permanent link">&para;</a></h2>
-<p>See <a href="https://github.com/argoproj/argo-workflows/blob/main/SECURITY.md">SECURITY.md</a>.</p>
-
-              
-
-<!-- Giscus -->
-<h2 id="__comments">Comments</h2>
-
-<script src="https://giscus.app/client.js"
-        data-repo="argoproj/argo-workflows"
-        data-repo-id="MDEwOlJlcG9zaXRvcnkxMDA5ODI0NDk="
-        data-category="General"
-        data-category-id="MDE4OkRpc2N1c3Npb25DYXRlZ29yeTMyMTA3OTY1"
-        data-mapping="pathname"
-        data-reactions-enabled="1"
-        data-emit-metadata="0"
-        data-input-position="bottom"
-        data-theme="light"
-        data-lang="en"
-        data-loading="lazy"
-        crossorigin="anonymous"
-        async>
-</script>
 
+              <h1>Argo Workflows - The workflow engine for Kubernetes</h1>
```

Comment:
```suggestion
              <h1>Argo Workflows</h1>
```

To match the previous heading, on [line 4037 above](https://github.com/argoproj/argo-workflows/pull/12362/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L4037)